Sample Applications/WPFGallery/Models/User.cs

```diff
@@ -3,67 +3,184 @@
 using System.Linq;
 using System.Text;
 using System.Threading.Tasks;
+using System.ComponentModel;
 
-namespace WPFGallery.Models;
-
-public class User
```

Comment:
I think we can use ObservableObject here as well. The same pattern that we follow for view models and mark rest of the properties where we are calling OnPropertyChanged as ObservableProperty.